src/main/java/graphql/schema/idl/SchemaDirectiveWiringSchemaGeneratorPostProcessing.java

```diff
@@ -43,23 +43,35 @@ public SchemaDirectiveWiringSchemaGeneratorPostProcessing(TypeDefinitionRegistry
 
     @Override
     public GraphQLSchema process(GraphQLSchema originalSchema) {
-        GraphQLSchema newSchema = SchemaTransformer.transformSchema(originalSchema, new Visitor());
-        return newSchema.transform(builder -> {
-            // they could have changed the code registry so rebuild it
-            GraphQLCodeRegistry codeRegistry = this.codeRegistryBuilder.build();
-            builder.codeRegistry(codeRegistry);
-        });
```

Comment:
We  have two schema transformers here - one for a visitor that might not have done anything

And one because they MAY have changed the code registry.

Another way to fix this would be to have a way to change a schema's code registry safely (since its really a object swap in not a tree rebuild - but thats not this does

Reply:
This turned up in JProfiler as a heavily invoked bit of code in schema builds